src/libsync/account.cpp

```diff
@@ -515,4 +515,26 @@ void Account::setServerVersion(const QString &version)
     emit serverVersionChanged(this, oldServerVersion, version);
 }
 
+QString Account::defaultSyncRoot() const
+{
+    Q_ASSERT(!_defaultSyncRoot.isEmpty());
```

Comment:
Perhaps we should check whether the directory exists here, too. It might have been removed in the meantime.

Reply:
I think the user should take care.